src/infrastructure/jira/jira-client/jira-client.ts

```diff
@@ -62,6 +65,24 @@ class JiraClient {
 		return response.data;
 	};
 
+	deleteDesign = async (
+		designId: FigmaDesignIdentity,
+		connectInstallation: ConnectInstallation,
+	): Promise<FigmaDesignIdentity> => {
+		const url = new URL(
+			`/rest/designs/1.0/design/${designId.toAtlassianDesignId()}`,
```

Comment:
@damoster is this the correct endpoint for deleting designs from DD?

Reply:
Yes that's correct. Technically that's the public endpoint you've used here which connect apps call (i.e. you've done the right there but there's an internal equivalent for when internal services call DD such as the gdrive service)